internal/bminventory/inventory.go

```diff
@@ -393,6 +394,13 @@ func (b *bareMetalInventory) setDefaultRegisterClusterParams(_ context.Context,
 		params.NewClusterParams.HTTPSProxy = params.NewClusterParams.HTTPProxy
 	}
 
+	if params.NewClusterParams.DiskEncryption == nil {
```

Comment:
Shouldn't ``disk_encryption`` be nullable in the ``cluster-create-params`` (in the swagge) for it to be possible?